src/GraphQLController.php

```diff
@@ -45,7 +45,10 @@ public function query(Request $request, string $schema = null): JsonResponse
 
         // Complete each query in order
         foreach ($inputs as $input) {
-            $completedQueries[] = $this->executeQuery($schema, $input);
+            $completedQueries[] = $this->executeQuery(
+                $schema,
+                ($input['query'] ?? null) ? $input : ['query' => '']
+            );
```

Comment:
I find this code quite hard to read, do you think this one will work too?
```suggestion
            $input['query'] = $input['query'] ?? '';
            $completedQueries[] = $this->executeQuery($schema, $input);
```
In the future this might get promoted to `??=` once 7.4 ever reaches the baseline.

Reply:
Indeed, was trying too hard not to use `??=` ! 
I now moved the check to executeQuery().